app/assets/images/icons/icon-facebook.svg

```diff
@@ -1,5 +1,841 @@
-<svg xmlns="http://www.w3.org/2000/svg" width="13" height="23" viewBox="0 0 13 23">
-    <text fill="#CC9543" fill-rule="evenodd" font-family="FontAwesome" font-size="24" letter-spacing=".23" transform="translate(-148 -1)">
-        <tspan x="147.694" y="22"></tspan>
-    </text>
-</svg>
+
+
+
+
+
+
+<!DOCTYPE html>
```

Comment:
This file doesn't look right @105ron, it looks like the source code for the home page of the site?